flycheck.el

```diff
@@ -707,6 +707,13 @@ display all errors from other files."
   :safe #'flycheck-error-level-p
   :package-version '(flycheck . "32"))
 
+(defcustom flycheck-relevant-error-other-file-show t
+  "Whether to show errors from other files."
+  :group 'flycheck
+  :type 'boolean
+  :package-version '(flycheck . "32")
+  :safe #'booleanp)
```

Comment:
I wonder if a `-p` suffix would be appropriate, since this is a boolean flag.  Additionally, the name is a bit of a mouthful, but maybe that's justified by the consistency with `-minimum-level`?

Reply:
Another option could be to have add a new level to `flycheck-relevant-error-other-file-minimum-level` called `'never`.

I'm not totally sure it should have that `-p` suffix. I can definitely be wrong but I thought a predicate was only a function or here it's variable, right?

Reply:
> Another option could be to have add a new level to flycheck-relevant-error-other-file-minimum-level called 'never. Having only one option could avoid confusion.

That could work, but it will conflict if someone defines an error level called never.  That would be weird, of course.

> Btw, I'm not totally sure it should have that -p suffix. I can definitely be wrong but I thought a predicate was only a function while here it's variable, right?

No, I think you're right, actually :)

Reply:
And what about setting `flycheck-relevant-error-other-file-minimum-level` to nil means never show errors from other files? I think that would make sense.

Otherwise I'm totally up to renaming `flycheck-relevant-error-other-file-show` to something else. I think we should keep the `flycheck-relevant-error-other-file` prefix, to make it obvious it's "paired" with `flycheck-relevant-error-other-file-minimum-level`. I'm not the best at naming things though.

Reply:
> And what about setting flycheck-relevant-error-other-file-minimum-level to nil means never show errors from other files? I think that would make sense.

It could, but I think that value is already used to display all errors from other files.    Let's go with the original name.  I'll merge as soon as CI is back on track.